en/advanced/computer_vision.md

```diff
@@ -0,0 +1,32 @@
+# Computer Vision (VIO, Avoidance)
+
+[Computer vision](https://en.wikipedia.org/wiki/Computer_vision) techniques enable computers to use visual data to make sense of their environment. 
+
+PX4 uses computer vision systems (primarily running on [Companion Computers](../companion_computer/pixhawk_companion.md)) in order to support the following features:
+- [Optical Flow](https://docs.px4.io/en/sensor/optical_flow.html) uses a downward facing camera and a downward facing distance sensor for position estimation.
+- [Visual Inertial Odometry](#vio) is used for navigation without a global position source (e.g. indoors).
+- [Obstacle Avoidance](https://docs.px4.io/en/computer_vision/obstacle_avoidance.html) provides navigation around obstacles when flying a planned path (currently missions are supported). This uses [PX4/avoidance](https://github.com/PX4/avoidance) running on a companion computer.
+- [Collision Prevention](https://docs.px4.io/en/computer_vision/collision_prevention.html) is used to stop vehicles before they can crash into an obstacle (primarily when flying in manual modes).
+
+
+## Visual Inertial Odometry (VIO) {#vio}
+
+VIO is used for estimating the *pose* (position and orientation) of a moving vehicle relative to a *local* starting position.
+This can be used to navigate a vehicle in the absence of GPS (or another global position source), and is used, in particular, for indoor navigation.
```

Comment:
You can add that is also helps outdoors, in situations where you lose GPS, like under a bridge.